lib/board.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class Board
+  def initialize(layout:)
+    @layout = layout
+  end
+
+  def is_full(tokens:)
```

Comment:
Feels a bit weird that you have to pass the tokens to this method when the `Board` already implicitly knows about tokens.

Reply:
Agree.